intel_extension_for_transformers/llm/runtime/graph/core/ne_layers.c

```diff
@@ -130,7 +130,7 @@ static_assert(sizeof(block_q8_1) == 2 * sizeof(float) + QK8_1, "wrong q8_1 block
 #if UINTPTR_MAX == 0xFFFFFFFF
 #define NE_MEM_ALIGN 4
 #else
-#define NE_MEM_ALIGN 16
+#define NE_MEM_ALIGN 64
```

Comment:
why we change mem align from 16 to 64 ?

Reply:
16-byte is not useful to Intel CPUs. Always padded to 64 bytes, and no need to consider 32-bit system which our kernel does not support.